core/test/regression/importer/importer_spec.js

```diff
@@ -116,15 +116,14 @@ describe('Integration: Importer', function () {
 
     beforeEach(function () {
         sinon.stub(importer, 'cleanUp');
+        should.exist(importer);
```

Comment:
this `should.exist(importer);` statement doesn't make sense :sweat_smile: something to be cleaned up after merge I think.